roles/taiga-node/tasks/main.yml

```diff
@@ -23,7 +23,7 @@
   become: true
   become_user: root
   apt_repository:
-    repo: "deb https://deb.nodesource.com/node_8.x {{ ansible_distribution_release }} main"
+    repo: "deb https://deb.nodesource.com/node_{{ taiga_node_version | default('8') }}.x {{ ansible_distribution_release }} main"
```

Comment:
Please don't use a `| default()` filter here. Instead, add the default in `taiga-node/defaults/main.yml`, then fix up `generate-variables-md.sh` to include the `taiga-node` role, and finally run that script so that it updates `variables.md`. Thanks!

Reply:
done